packages/i18n/en.json

```diff
@@ -40,6 +40,10 @@
     }
   },
   "pages": {
+    "404": {
+      "title": "Whooops ?!… sounds you're lost… Sorry. Really.",
+      "link": "Please, please… come back home."
+    },
```

Comment:
Could you make also the translation for `fr.json` file ?

Reply:
done :)